shell/pages/c/_cluster/fleet/index.vue

```diff
@@ -284,223 +425,510 @@ export default {
+                class="btn role-primary"
+              >
+                {{ t('fleet.gitRepo.intro.add') }}
+              </router-link>
+            </div>
+          </div>
+          <div
+            v-if="viewMode === 'cards'"
+            class="cards-panel"
+          >
+            <div
+              v-for="(state, j) in applicationStates[workspace.id]"
+              :key="j"
+              :data-testid="`state-panel-${ state.stateDisplay }`"
+            >
+              <div
+                v-if="cardResources[workspace.id][state.stateDisplay]?.length"
+                class="card-panel"
+              >
+                <div
+                  role="button"
+                  tabindex="0"
+                  class="title"
+                  :aria-label="t(`${ isStateCollapsed[workspace.id]?.[state.stateDisplay] ? 'expand': 'collapse' }-${ state.stateDisplay }`)"
+                  @click="toggleState(workspace.id, state.stateDisplay)"
+                  @keydown.space.enter.stop.prevent="toggleState(workspace.id, state.stateDisplay)"
+                >
+                  <i
+                    :class="{
+                      ['icon icon-chevron-right']: isStateCollapsed[workspace.id]?.[state.stateDisplay],
+                      ['icon icon-chevron-down']: !isStateCollapsed[workspace.id]?.[state.stateDisplay],
+                    }"
+                  />
+                  <i
+                    v-if="state.statePanel.id !== 'success'"
+                    class="ml-5 state-icon"
+                    :class="state.statePanel.icon"
+                    :style="{ color: state.statePanel.color }"
+                  />
+                  <div class="label">
+                    <span class="partial">
+                      {{ state.stateDisplay }}&nbsp;&nbsp;{{ cardResources[workspace.id]?.[state.stateDisplay]?.length }}
+                    </span>
+                    <span class="total label-secondary">/{{ workspace.repos.length }}</span>
+                  </div>
+                </div>
+                <div
+                  v-if="!isStateCollapsed[workspace.id]?.[state.stateDisplay]"
+                  class="card-panel-body"
+                >
+                  <div class="resource-cards-container">
+                    <div
+                      v-for="(item, y) in cardResources[workspace.id][state.stateDisplay]"
+                      :key="y"
+                      class="resource-card"
+                      :class="{
+                        ['selected']: selectedCard === `${ item.id }-${ y }` && isOpenSlideInPanel
+                      }"
+                      :data-testid="`card-${ item.id }`"
+                    >
+                      <ResourceCard
+                        v-if="y < (cardsCount[workspace.id]?.[state.stateDisplay] || CARDS_MIN)"
+                        role="button"
+                        tabindex="0"
+                        :aria-label="`aria-label-card-${ item.id }`"
```

Comment:
This needs a descriptive label that's easily relayed via assistive technologies.